go/database/mpt/archive_trie.go

```diff
@@ -549,6 +551,11 @@ func (a *ArchiveTrie) addError(err error) error {
 	return a.archiveError
 }
 
+// Directory returns the directory where the archive is stored on disk.
+func (a *ArchiveTrie) Directory() string {
```

Comment:
```suggestion
// GetDirectory returns the path to directory where the archive is stored on disk.
func (a *ArchiveTrie) GetDirectory() string {
```

Reply:
It is idiomatic (albeit not followed everywhere) to name getters without `Get..()`  in golang: https://go.dev/doc/effective_go#Getters